pandokia/import_data.py

```diff
@@ -476,7 +501,15 @@ def run(argv, hack_callback=None):
                     print('Skipped: {}'.format(rx.test_name))
                 duplicate_count += 1
 
-            pdk_db.commit()
+            pdk_db.commit() 
+        
+        if reimport_count != 0:
+            print("There are {:d} reimport tests during this run".format(reimport_count))
```

Comment:
Can we please change this message to "{:d} tests were re-imported during this run"
and the subsequent one to "See details in %s" ?

Reply:
Will do.